apps/condo/bin/generate-miniapps.js

```diff
@@ -0,0 +1,89 @@
+const path = require('path')
```

Comment:
Could you please add documentation and usage examples?

Reply:
I'll do...but a bit later